src/app/services/chart.service.ts

```diff
@@ -17,7 +17,6 @@ export class ChartService {
 
     private timeOptions = {
           'yearly': '%Y',
-          'daily': '%Y-%m-%d',
```

Comment:
Should there be a quarterly format added here?

Reply:
Hmm, we don't use it but it wouldn't hurt to be thorough

Reply:
Actually, can't do this now. It would require additional refactoring. In the quarterly case, we need to convert the keys in the API (e.g. 2010-3) into the start date for a "quarter" which would be July 1, 2010 for 2010-3, and that can't be done with `D3.timeParse`. We'd need this refactor for pretty much all of the other remaining valid time aggs if we chose to display them in the lab UI.